docs/templates/template-def-v1.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: page
+title: Version 1
+permalink: /template-version-1/
+parent: Templates
+---
+
+# Template Definition Version 1 <sup>depricated</sup>
+
+The following objects can be used in template files.
+
+### **project**
+
+General information about the project are stored.
+
+
+| Name        | Description     |      Example  |
+| ------------- |-------------| -------------|
+| projecthome   | home directory | `${projecthome}` |
+| project.targetPath | directory where the template is checked out  | `${projecthome}/${project.targetPath}/generator-templates/entity/entity.ftl` |
+| project.title | name of the project | `${projecthome}/${project.targetPath}/${project.title}/persistence/` |
+| project.template | project template used - not used in templates |  |
+| project.packagePrefix | you can choose a package name which will be included in the Java-package path | `${projecthome}/${project.targetPath}/${project.title}/persistence/src/main/java/de/${project.packagePrefix?lower_case}/${project.title?lower_case}/entity/`
+| project.description | description of the project which is not used in templates warning: not possible anymore | |
+| project.domains | domain objects.   | |
+
+### **domain**
+
+The domain object is normally used in the template-files (*.ftl).
+
+#### Example
+```
+@Table(name="${domain.name?upper_case}")
+public class ${domain.name}Entity extends AbstractEntity {
+
+	private static final long serialVersionUID = 1L;
+
+	//domain attributes
+	<#list (domain.getAttributes()) as attribute>
+		<#if attribute.pattern?? && attribute.pattern?length &gt; 0 && attribute.dataType == "String">
+	@Pattern(regexp = "${attribute.pattern}")
+		</#if>
+		<#if attribute.dataType == "String">
+			<#if !attribute.nullable>
+```
+
+#### Properties
+
+| Name        | Description     |      Example  |
```

Comment:
The Data-Type is missing here

Reply:
are placed above

Reply:
it's deprecated - I just want to let it as it was before